cli/flags.rs

```diff
@@ -86,6 +88,11 @@ pub struct FmtFlags {
   pub files: Vec<PathBuf>,
   pub ignore: Vec<PathBuf>,
   pub ext: String,
+  pub use_tabs: Option<bool>,
+  pub line_width: Option<NonZeroU32>,
+  pub indent_width: Option<NonZeroU8>,
+  pub single_quote: Option<bool>,
+  pub prose_wrap: Option<String>,
```

Comment:
These `Option<bool>` might be an overkill, but I find it easier to reason about when we need to overwrite `FmtConfigOptions`